applications/MeshMovingApplication/python_scripts/trilinos_mesh_solver_base.py

```diff
@@ -51,11 +51,6 @@ def get_communicator(self):
             self._communicator = TrilinosApplication.CreateCommunicator()
         return self._communicator
 
-    def print_on_rank_zero(self, *args):
-        KratosMPI.mpi.world.barrier()
```

Comment:
there is now no more barrier but I don't think it will create a problem

What do you think @jcotela ?

Reply:
There was a barrier there? :scream: 
Good riddance!